holoviews/streams.py

```diff
@@ -75,7 +75,9 @@ class Stream(param.Parameterized):
     """
 
     # Mapping from uuid to stream instance
-    registry = OrderedDict()
+    registry = defaultdict(list)
+
+    _callbacks = defaultdict(dict)
```

Comment:
Yea definitely need to add docstrings in this file still. Basically it defines the registry of callbacks for each backend which supply the stream information.